sauce/features/budget/check-credit-balances/index.js

```diff
@@ -71,9 +78,25 @@ export class CheckCreditBalances extends Feature {
     return debtAccounts || [];
   }
 
+  getCheckCardbalancesMode() {
+    switch (ynabToolKit.options.CheckCreditBalances) {
```

Comment:
You can also use this.settings.enabled to get the users choice. I prefer that approach as it doesn't tie you to the name of the feature.